Users/views.py

```diff
@@ -77,3 +76,48 @@ def register(request):
             # else: 
             #     return render(request,'registration/login.html',{})
     else: return render(request,'registration/login.html',{})
+
+def forgot_pass(request):
+    global user_name
+    if request.method == 'POST': 
+        user_name = request.POST.get('username')
+        
+        user = User.objects.all().filter(username=user_name)
+        if user:
+            global verify_code
```

Comment:
Are you sure this global username and verify_code will work when more than 1 lets say 10 users are simultaneously resetting their password? I don't think so :(

These global variables will become inconsistence, 
like first user1 comes and verify_code is set to "xyz" then before user1 get verified if user2 comes and sets verify_code to "abc", What now? user1 will fail to get verified.

BTW django provides built in methods for password reset and and change and someone has already added password reset feature here #80 , Great try though :smile:

You will get points for all your efforts :+1:     
If you can fix this and make it work without using built in things, I'll approve and merge your changes and award you with extra points :smile:

Reply:
Okay sir, i'll try other way out